packages/clients/test/aptos/wallet.integration.test.ts

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright 2023-2024 Shinami Corp.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { describe, expect, it } from "@jest/globals";
+import { v4 as uuidv4 } from "uuid";
+import { KeySession, ShinamiWalletSigner } from "../../src/aptos/index.js";
+import {
+  EXAMPLE_PACKAGE_ID,
+  createAptos,
+  createWalletClient,
+  createKeyClient,
+} from "./integration.env.js";
+import {
+  AccountAddress,
+  Deserializer,
+  AccountAuthenticator,
+  AccountAuthenticatorEd25519,
+  isUserTransactionResponse,
+} from "@aptos-labs/ts-sdk";
+
+const aptos = createAptos();
+const key = createKeyClient();
+const wal = createWalletClient();
+
+function delay(ms: number) {
```

Comment:
I usually avoid this in favor of more targeted waits. Is there a wait helper from Aptos sdk we can use or some other thing we can poll for?

Reply:
Couldn't find anything specific in aptos sdk, opted to poll on fetching account info in a retry loop